app/assets/stylesheets/application.scss

```diff
@@ -21,20 +21,25 @@ $govuk-global-styles: true;
 }
 
 .task-checkboxes {
-  .govuk-checkboxes__item {
-    margin-bottom: govuk-spacing(9);
-  }
-}
-
-.task-checkboxes,
-.clear-legal-documents-task-checkboxes {
   .govuk-checkboxes__item {
     label {
       @extend .govuk-\!-font-weight-bold;
     }
   }
 }
 
+.task-action__wrapper:first-of-type {
+  margin-top: 0;
+}
+
+.task-action__wrapper__padding-normal {
+  margin-top: govuk-spacing(9);
```

Comment:
I think this padding might be too much! Looking at the prototype, `4` looks closer

Reply:
Also I think the reduced padding should be 0, but you would have to set `.govuk-form-group: margin-bottom: 10px` as each checkbox is now its own group.